lesson_26/homework_26.py

```diff
@@ -0,0 +1,80 @@
+"""Hometask 26."""
+
+from importlib import import_module
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+"""
+Написати на python selenium код який пройде по двох фреймах
+на початковiй сторiнцi, ввійде в кожний фрейм, введе правильний
+секретний текст, натисне кнопку “Перевiрити”, порівняє текст
+дiалогового вiкна для підтвердження успішності верифікації
+та закриє діалогове вікно.
+"""
+
+
+class SecretVerification:
+    """Class for verifying secrets inside frames of a webpage."""
+
+    def __init__(self, url, browser, success_message):
+        """Initialize params."""
+        self.url = url
+        self.driver = self.driver_initialize(browser)
+        self.success_message = success_message
+
+    def driver_initialize(self, browser):
+        """Initialize the webdriver based on the chosen browser."""
+        try:
+            # Dynamic browser module import
+            webdriver_module = import_module(
+                f'selenium.webdriver.{browser.lower()}.webdriver')
+            driver_class = getattr(webdriver_module, 'WebDriver')
```

Comment:
possible option but better to do this via pytest. BTW - testing better to organize for future tasks as pytest + selenium or pytest + PlayWright